frontend/web/components/LicensingTabContent.tsx

```diff
@@ -0,0 +1,110 @@
+import { useUploadOrganisationLicenceMutation } from 'common/services/useOrganisationLicensing'
+import React, { useEffect, useRef, useState } from 'react'
+import Button from './base/forms/Button'
+import Utils from 'common/utils/utils'
+
+type LicensingTabContentProps = {
+  organisationId: number
+}
+
+const LicensingTabContent: React.FC<LicensingTabContentProps> = ({
+  organisationId,
+}) => {
+  const [uploadOrganisationLicence, { error, isLoading, isSuccess }] =
+    useUploadOrganisationLicenceMutation()
+
+  const [licence, setLicence] = useState<File | null>(null)
+  const [licenceSignature, setLicenceSignature] = useState<File | null>(null)
+
+  const licenceInputRef = useRef<HTMLInputElement>(null)
+  const licenceSignatureInputRef = useRef<HTMLInputElement>(null)
+
+  useEffect(() => {
+    if (isSuccess) {
+      toast('Licence uploaded successfully')
+    }
+
+    if (!isSuccess && error?.data) {
+      toast(
+        Array.isArray(error?.data)
+          ? error?.data[0]
+          : 'Upload was not successfull',
```

Comment:
```suggestion
          : 'Upload was not successful',
```

Reply:
It might be worth checking that you have a spellchecker enabled in your IDE to ensure we don't end up with typos like this.